tests/core/CoreIndexFile.spec.ts

```diff
@@ -41,19 +41,19 @@ describe('CoreIndexFile', async () => {
       expect(parsedCoreIndexFile.recoverDidSuffixes[0]).toEqual(recoverOperation.didUniqueSuffix);
       expect(parsedCoreIndexFile.deactivateDidSuffixes.length).toEqual(1);
       expect(parsedCoreIndexFile.deactivateDidSuffixes[0]).toEqual(deactivateOperation.didUniqueSuffix);
-      expect(parsedCoreIndexFile.model.provisionalIndexFileUri).toEqual(provisionalIndexFileUri);
+      expect(parsedCoreIndexFile.model.coreIndexFileUri).toEqual(coreIndexFileUri);
     });
 
     it('should throw error if core proof file is specified but there is no recover and no deactivate operation.', async () => {
-      const provisionalIndexFileUri = 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA';
+      const coreIndexFileUri = 'EiB4ypIXxG9aFhXv2YC8I2tQvLEBbQAsNzHmph17vMfVYA';
       const coreProofFileUri = 'EiBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB'; // Should not be allowed with no recovers and deactivates.
 
       // Create operation.
       const createOperationData = await OperationGenerator.generateCreateOperation();
       const createOperation = createOperationData.createOperation;
 
       const coreIndexFileBuffer =
-        await CoreIndexFile.createBuffer(undefined, provisionalIndexFileUri, coreProofFileUri, [createOperation], [], []);
+        await CoreIndexFile.createBuffer(undefined, coreIndexFileUri, coreProofFileUri, [createOperation], [], []);
```

Comment:
Sanity: I think you are incorrectly changing these, `provisonalIndexFileUri` is correct.

Reply:
Reverted, it was a mistake.